src/main/java/backend/resource/TurboIssue.java

```diff
@@ -338,6 +339,28 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return Optional.empty();
     }
 
+    /**
+     * Matching is done by matching all words separated by space in query
```

Comment:
Rather, each new word potentially brings the target issue closer to the top of the search results. We cannot expect the user to remember the exact words precisely. If you use `AND`, one tiny typo will make the issue impossible to locate. It is fine to use `AND` now, but do create another issue to change it to a smarter search algo.